lxd/containers.go

```diff
@@ -2165,6 +2167,8 @@ func containerExecPost(d *Daemon, r *http.Request) Response {
 	if post.WaitForWS {
 		ws := &execWs{}
 		ws.fds = map[int]string{}
+		ws.rootUid = int(d.idMap.Uidmin)
```

Comment:
I don't think so.  We can, but it won't buy us anything.  It just means the fchown may be a noop.  Complicating our code to avoid the noop doesn't seem worthwhile, unless there is some actual error I'm not being imaginative enough to think up.